### DISCUSSION TITLE
Is it possible to use Jellyseer with this plugin ?

### DISCUSSION THREAD

**Discussion by traxys** [9p4/jellyfin-plugin-sso#132] [Q&A]
[Jellyseer](https://github.com/Fallenbagel/jellyseerr) allows you to use your jellyfin account to connect to the instance, but I'm not really sure if it's possible to link to users that have been registered with the SSO plugin ?

**Comment by 9p4** [comment]
afaik it *should* work. I don't have a computer to test with for the next week or so, so I can't really test it out and see if it works. would you be willing to try it out and let me know if it works?

**Reply by traxys** [reply]
Using jellyseer you need to provide (in jellyseer) the `username` and `password`, but that's not really possible when using SSO. There's a work around consisting of click "password reset" and creating a password for the user, but that's not the nicest

**Reply by 9p4** [reply]
You can set the fallback authentication provider in the SSO plugin so you don't have to go through the reset process.

**Reply by traxys** [reply]
I'm not sure what you mean, the ' Set default Provider:' to 'Jellyfin.Server.Implementations.Users.DefaultAuthenticationProvider' ?

**Reply by 9p4** [reply]
Yes, that should help you with logging in with a fallback password for the users.
